<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VatsinatorWindow</class>
 <widget class="QMainWindow" name="VatsinatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vatsinator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/icons.qrc">
    <normaloff>:/icons/vatsinator.png</normaloff>:/icons/vatsinator.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="Vatsinator::Widgets::MapWidget" name="map" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVatsinator">
    <property name="title">
     <string>&amp;Vatsinator</string>
    </property>
    <addaction name="actionMetars"/>
    <addaction name="actionClients"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settin&amp;gs</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuVatsinator"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/vatsinator.png</normaloff>:/icons/vatsinator.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionMetars">
   <property name="text">
    <string>&amp;METARs...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionClients">
   <property name="text">
    <string>Clients...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Vatsinator::Widgets::MapWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/mapwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>VatsinatorWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
